test/parallel/test-repl-history-navigation.js

```diff
@@ -554,6 +555,32 @@ const tests = [
     expected: [],
     clean: false
   },
+  {
+    env: { NODE_REPL_HISTORY: defaultHistoryPath },
+    test: ['const util = {}', ENTER,
+           'ut', RIGHT, ENTER],
+    expected: [
+      prompt, ...'const util = {}',
+      'undefined\n',
+      prompt, ...'ut', ' // il', '\n// {}', 'il',
+      '\n// {}',
+      prompt,
+    ],
+    clean: false
+  },
+  {
+    env: { NODE_REPL_HISTORY: defaultHistoryPath },
+    test: ['globalThis.util = {}', ENTER,
+           'ut', RIGHT, ENTER],
```

Comment:
I think you need to re-define the `util` global so following test cases do not fail.
```suggestion
    test: ['const utilDesc = Reflect.getOwnPropertyDescriptor(globalThis, "util")', ENTER,
           'globalThis.util = {}', ENTER,
           'ut', RIGHT, ENTER,
           'Reflect.defineProperty(globalThis, "util", utilDesc)', ENTER],
```